root/static/scripts/edit/URLCleanup.js

```diff
@@ -635,6 +635,17 @@ const CLEANUPS: CleanupEntries = {
         };
       }
 
+      if (/amazon\.com\/vdp\//i.test(url)) {
```

Comment:
To match video samples that exist in other Amazon stores too:

```suggestion
      if (/^https:\/\/www\.amazon\.[^\/]+\/vdp\//i.test(url)) {
```

Reply:
Oh, duh, fair point :)